### ISSUE TITLE
is there a XML editor as cool as this?

### ISSUE THREAD

**Issue by brean** [jdorn/json-editor#630]
Hi,

I am looking for a Web-Editor just like this that takes a XML-Schema and XML data and creates a nice editable form just like the json-editor. So is there something out there that is as cool as this json-editor for XML? I guess in theory it should be possible to fork the json-editor and create an xml-editor?

**Comment by mcundiff** [comment]
Yes, there is. It is called Xonomy. It does not work directly from an XML Schema but from a "docSpec" that you create that serves a similar purpose.

http://www.lexiconista.com/xonomy/

**Comment by brean** [comment]
Xonomy looks cool, but it does not generate a normal HTML form.
